src/modules/add-task.js

```diff
@@ -0,0 +1,41 @@
+// eslint-disable-next-line import/no-cycle
+import { renderList } from './todolist.js';
+
+const addToList = document.querySelector('#add-to-list');
+
+// eslint-disable-next-line import/no-mutable-exports
+let todoList = [];
+
+function saveData() {
+  localStorage.setItem('todolist', JSON.stringify(todoList));
+}
+
+function loadData() {
+  const savedData = JSON.parse(localStorage.getItem('todolist'));
+  if (savedData) {
+    todoList = savedData;
+  }
+}
+
+function addTask(event) {
+  event.preventDefault();
+  const taskData = {
+    description: '',
+    completed: false,
+    index: todoList.length,
```

Comment:
- [ ] As required, I kindly suggest deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. Currently your indexes remain the same even after deleing an item in the middle.
Kindly check the video below for more explanation

https://www.loom.com/share/c4b59e193c3f4022af05fcb14f805b6a

The number pointed out by the red arrow will be 2 if the indexes are updated after deleing:
<img width="752" alt="image" src="https://user-images.githubusercontent.com/52934128/230619579-a94f2648-6639-423e-b0de-4b0d7a5b2140.png">